src/collections/Breadcrumb/Breadcrumb.js

```diff
@@ -34,35 +34,17 @@ function Breadcrumb(props) {
   const rest = getUnhandledProps(Breadcrumb, props)
   const ElementType = getElementType(Breadcrumb, props)
 
-  if (!_.isNil(children)) {
-    return <ElementType {...rest} className={classes}>{children}</ElementType>
-  }
+  if (!_.isNil(children)) return <ElementType {...rest} className={classes}>{children}</ElementType>
 
   const childElements = []
 
   _.each(sections, (section, index) => {
     // section
-    const breadcrumbSection = BreadcrumbSection.create(section)
-    childElements.push(breadcrumbSection)
+    childElements.push(BreadcrumbSection.create(section))
 
     // divider
     if (index !== sections.length - 1) {
-      // TODO generate a key from breadcrumbSection.props once this is merged:
-      // https://github.com/Semantic-Org/Semantic-UI-React/pull/645
-      //
-      // Stringify the props of the section as the divider key.
-      //
-      // Section:     { content: 'Home', link: true, onClick: handleClick }
-      // Divider key: content=Home|link=true|onClick=handleClick
-      let key
-      if (section.key) {
-        key = `${section.key}_divider`
-      } else {
-        key = _.map(breadcrumbSection.props, (v, k) => {
-          return `${k}=${typeof v === 'function' ? v.name || 'func' : v}`
-        }).join('|')
-      }
-      childElements.push(BreadcrumbDivider.create({ content: divider, icon, key }))
```

Comment:
I misread this initially, we are one using the factory as an object in this case.  The key is then required by us, otherwise, we're getting this in the docs for Breadcrumb:

`Each child in an array or iterator should have a unique "key" prop. Check the top-level render call using <div>.`